resources/debugger/py_debug_wrapper.py

```diff
@@ -0,0 +1,14 @@
+import os
+import runpy
+import sys
+
+if __name__ == '__main__':
+    # debugger will be available in the task root if we patched the user's requirements.txt correctly
+    task_root = os.environ['LAMBDA_TASK_ROOT']
```

Comment:
This also resolve the `/var/lang/bin/python3.8: No module named ptvsd` issue that Justin was seeing